src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -116,7 +116,7 @@ protected function isPropertyEnabled(string $property): bool
      */
     protected function isPropertyMapped(string $property, string $resourceClass, bool $allowAssociation = false): bool
     {
-        if ($this->isPropertyNested($property)) {
+        if ($this->isPropertyNested($property) && !$this->isPropertyNestedEmbedded($property, $resourceClass)) {
```

Comment:
Yes it will breaks 👍 
If we put the code in isPropertyNested, it disturbs me for differents reasons:
- 1/ it means that a embed field is not considered as a nested field. In fact it is by definition :-)
- 2/ the function extractProperties in AbstractFilter use isPropertyNested and for an Embed field, it has to be needsFixing to true. Otherwise, it fails for the request.

This code is in AbstractFilter, so any Filter will benefit the code.
Normannly the only thing to change for each filter is isBooleanField, isDateField, isNumericField...